### DISCUSSION TITLE
How to add the required element at the end of the el-select-v2 dropdown box（如何在el-select-v2下拉框最后添加元素）

### DISCUSSION THREAD

**Discussion by tuskermanshu** [element-plus/element-plus#15195] [Q&A]
There is a requirement to implement dropdown loading functionality, and using el-select-v2 I found that there is no similar slot provided（有一个需求要实现下拉加载功能，使用el-select-v2发现没有类似的插槽提供）

**Comment by kooriookami** [comment]
It seems not a common behavior in PC web.
